api/src/twitterBot/bullQueue.ts

```diff
@@ -0,0 +1,60 @@
+import { Worker, Queue, QueueScheduler } from "bullmq";
+import createDebug from "debug";
+import Redis from "ioredis";
+import { format, subDays } from "date-fns";
+import { tweet } from "./twit";
+import { microToStacks, readData } from "../utils";
+
+const queueName = "tweet-stats";
+const debug = createDebug(`queue:${queueName}`);
+const redisClient = new Redis();
+
+export const tweetStatsQueueScheduler = new QueueScheduler(queueName, {
+  connection: redisClient,
+});
+export const tweetStatsQueue = new Queue<{}>(queueName, {
+  connection: redisClient,
+});
+
+const currentData = readData();
```

Comment:
this should be in the worker job to get the latest data when the job is triggered